deployment/packer/cac.json

```diff
@@ -0,0 +1,94 @@
+{
+  "variables": {
+    "local_project_directory": ".",
+    "aws_access_key": "",
```

Comment:
Interesting thought. However, those keys in cac_secrets are actually going to be removed very shortly when we switch to using IAM roles.